docgenhtml/plugins/org.polarsys.capella.docgen/templates/pattern._2twVYKu9EeCWrf9pgx3zjA/method._2twVZau9EeCWrf9pgx3zjA.pt

```diff
@@ -21,9 +21,9 @@ String logo = ImageHelper.getTypePng(element, projectName, outputFolder);
 <%
 org.polarsys.kitalpha.doc.gen.business.core.extension.intf.DocGenExtensionFactory.newDocGenExtensionEngine().apply("CAPELLA_REQUIREMENTS_EXTENSION", ctx, getParameters(), stringBuffer);
 %>
-
-
-
+<%

```

Comment:
Is this a cherrypick of a forgotten commit?

Reply:
This is a cherry pick from a commit done on master (https://github.com/Iulius3456/capella-xhtml-docgen/commit/1f65575f80d92a9fcf88e308831e46682c4b2ca7)